TFlippy_TerritoryControl_Weapons_Dev/Entities/Weapons/Fragmine/FragMine.as

```diff
@@ -144,7 +144,7 @@ bool explodeOnCollideWithBlob(CBlob@ this, CBlob@ blob)
 
 bool doesCollideWithBlob(CBlob@ this, CBlob@ blob)
 {
-	return blob.getShape().isStatic() && blob.isCollidable();
+	return blob.getShape().isStatic() && blob.isCollidable() blob.getName() == "fragmine";
```

Comment:
This is missing an operator and will error! I assume you can see what is missing.